config.m4

```diff
@@ -313,6 +313,7 @@ PHP_ARG_WITH(libbson, Use system libbson,
     fi
     PHP_EVAL_INCLINE($LIBBSON_INC)
     PHP_EVAL_LIBLINE($LIBBSON_LIB, MONGODB_SHARED_LIBADD)
+    AC_DEFINE(HAVE_LIBBSON, 1, [Use system libbson])
```

Comment:
As to not make this confusing, perhaps we should use HAVE_SYSTEM_LIBBSON and HAVE_SYSTEM_LIBMONGOC? Because, we're always going to have a LIBBSON and LIBMONGOC.